client/my-sites/site-settings/sitemaps.jsx

```diff
@@ -0,0 +1,196 @@
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
+import JetpackModuleToggle from 'my-sites/site-settings/jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import QueryJetpackConnection from 'components/data/query-jetpack-connection';
+import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import { getJetpackModule, isActivatingJetpackModule, isJetpackModuleActive } from 'state/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class Sitemaps extends Component {
+	static defaultProps = {
+		isSavingSettings: false,
+		isRequestingSettings: true,
+	};
+
+	static propTypes = {
+		isSavingSettings: PropTypes.bool,
+		isRequestingSettings: PropTypes.bool,
+	};
+
+	renderSitemapLink( sitemapUrl ) {
+		return (
+			<div key={ sitemapUrl }>
+				<ExternalLink href={ sitemapUrl } icon target="_blank">
+					{ sitemapUrl }
+				</ExternalLink>
+			</div>
+		);
+	}
+
+	renderInfoLink( url ) {
+		const { translate } = this.props;
+
+		return (
+			<div className="sitemaps__info-link-container site-settings__info-link-container">
+				<InfoPopover position={ 'left' }>
+					<ExternalLink href={ url } icon target="_blank">
+						{ translate( 'Learn more about Sitemaps.' ) }
+					</ExternalLink>
+				</InfoPopover>
+			</div>
+		);
+	}
+
+	renderSitemapExplanation() {
+		const { translate } = this.props;
+
+		return (
+			<FormSettingExplanation>
+				{ translate( 'Your sitemap is automatically sent to all major search engines for indexing.' ) }
+			</FormSettingExplanation>
+		);
+	}
+
+	renderWpcomSettings() {
+		const { site } = this.props;
+		const sitemapTypes = [
+			'sitemap',
+			'news-sitemap',
+		];
+
+		return (
+			<div>
+				{ this.renderInfoLink( 'https://support.wordpress.com/sitemaps/' ) }
+
+				{ this.renderSitemapExplanation() }
+
+				{
+					sitemapTypes.map( ( sitemapType ) => {
+						const sitemapUrl = site.URL + '/' + sitemapType + '.xml';
+						return this.renderSitemapLink( sitemapUrl );
+					} )
+				}
+			</div>
+		);
+	}
+
+	renderJetpackSettingsContent() {
+		const {
+			activatingSitemapsModule,
+			sitemapsModule,
+			sitemapsModuleActive,
+			translate
+		} = this.props;
+
+		if ( ! activatingSitemapsModule && ! sitemapsModuleActive ) {
+			return;
+		}
+
+		const sitemapTypes = [
+			'sitemap_url',
+			'news_sitemap_url',
+		];
+
+		return (
+			<div className="sitemaps__module-settings site-settings__child-settings">
+				{
+					activatingSitemapsModule &&
+					<FormSettingExplanation>
+						{ translate( 'Generating sitemap…' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Generating...' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/site-settings-sitemaps-card)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).